README.md

```diff
@@ -87,6 +88,25 @@ npm config set save-exact
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Specify engines on `package.json`
+
+### Why?
+
+Specifing minimal engines version for your module warns the user if he is not using a supported version. This is specially important for ensuring npm3 flat tree dependecy on Windows, or es2015 features that your tasks require on node.
```

Comment:
**Minimal vs exact**

Should we make a distinction between "minimal" and "exact" engines? The example under *How?* uses exact ones. I think that's best as that assures the whole team has the same issues (recalling path resolve issue in 5.7.0). So maybe just "Specifying engine versions"?

**Prerelease versions**

Should we note [setting engine versions means pre release versions will not be allowed](https://github.com/npm/npm/issues/7171)? With exact versions its something we want to prevent anyway. I'd say if someone wants to use a prerelease do so by changing it in the `package.json` or temporarily removing the `engines` property. 

**Spelling**

"engines version" -> "engine versions"
"npm3" -> "npm 3" or "npm@3" ([npm itself uses these two variations](http://blog.npmjs.org/post/122450408965/npm-weekly-20-npm-3-is-here-ish))
"dependecy" -> "depende**n**cy"
"es2015" -> "ES2015"